pkg/occlient/occlient.go

```diff
@@ -1098,38 +1096,15 @@ func (c *Client) DeleteProject(name string) error {
 // GetLabelValues get label values of given label from objects in project that are matching selector
 // returns slice of uniq label values
 func (c *Client) GetLabelValues(project string, label string, selector string) ([]string, error) {
-	// get all object that have given label
-	// and show just label values separated by ,
-	args := []string{
-		"get", "all",
-		"--selector", selector,
-		"--namespace", project,
-		"-o", "go-template={{range .items}}{{range $key, $value := .metadata.labels}}{{if eq $key \"" + label + "\"}}{{$value}},{{end}}{{end}}{{end}}",
-	}
-
-	output, err := c.runOcComamnd(&OcCommand{args: args})
-	if err != nil {
-		return nil, err
-	}
-
+	appLabel, _ := c.appsClient.DeploymentConfigs(project).List(metav1.ListOptions{LabelSelector: selector})
```

Comment:
check for error!

Reply:
why is variable called `appLabel`? When it is containing List of DeploymentConfigs.